HyperPlay-Gaming/hyperplay-desktop-client
# feat: send game update notifications

<--- Put the description here --->
How to test:
- Have in the library a game with an available update or change the `library.json` file in the app data.
- Run the app
- You should see the notifications pop up
---

Use the following Checklist if you have changed something on the Backend or Frontend:
